tests/jdk/map_check.rs

```diff
@@ -55,4 +121,15 @@ fn everything() {
     t1(&map, &keys[..], SIZE);
     // get (absent)
     t1(&map, &absent_keys[..], 0);
+    // contains_key (present & absent)
```

Comment:
Can you place these in the same order as they are in the original Java tests? Just to make referencing those files easier in the future?

Reply:
Yep, done.